extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -35,8 +35,8 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 			$win = $(win);
 
 		function getStartPosition(placeHolder) {
-			return parseInt(placeHolder.offset().top, 10) +
-				parseInt(placeHolder.height(), 10) -
+			return parseInt(placeHolder.offset().top, 10) -
+					// TODO understand when palceholder height is required
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)